config.py

```diff
@@ -27,13 +27,21 @@
     print(f"Couldn't find 'database' section in '{CONFIG_FILE_PATH}'!")
     sys.exit(1)
 
+# Error if database type is not found
+if not config.has_option("database", "database_type"):
+    # Config file is likely an older version
+    print(
+        "Config file contains incorrect variable names. Please update your configuration to the correct format."
```

Comment:
@IanMayo Do we want a more specific message here that warns the user that they may be using an out-dated version of Pepys and to talk to their system administrator?

Reply:
Yes, this is a _special_ error that I'm happy to provide additional guidance to.

Let's go for:
`Config file contains variable names used in legacy versions of Pepys that provided insufficient support for database migration.  Please contact your system administrator to discuss renaming `db_xxx` to `database_xxx`.